src/Commands/ClassStartCommand.ts

```diff
@@ -35,9 +36,21 @@ export class ClassStartCommand extends Command {
         line += this.language.properties.classes.declareStartLeft;
         line += this.context.convertCommon("type", parameters[1]);
 
-        if (parameters.length === 3) {
-            line += this.language.properties.classes.declareExtendsLeft;
-            line += this.context.convertCommon("type", parameters[2]);
+        if (parameters.length === 4) {
+
+            if (parameters[2] === "implements") {
+
+                if (this.language.properties.interfaces.supported === false) {
+                    return LineResults.newSingleLine("", false);
```

Comment:
The class should still exist. Interfaces not existing just means we won't have it declared as inheriting from them.